### ISSUE TITLE
Camera Permission

### ISSUE THREAD

**Issue by trailseeker** [singpass/iOS-Singpass-in-app-browser-login-demo#4]
Why we need camera permission ?

**Comment by xun-da-gt** [comment]
There is an option where the user is able to password login which will require camera permission to allow face scanning for  Singpass Face Verification as multi-factor authentication.

**Comment by zhongshan1993** [comment]
@xun-da-gt But after we apply for camera permission in RP App, we still can't control whether the browser has camera permission, right?

**Comment by chinweekoh** [comment]
@zhongshan1993 correct, the app has no control over the browser's camera permission. The info.plist camera permission would be require if your app uses the camera function.
